sdks/python/apache_beam/io/bigquery_test.py

```diff
@@ -111,6 +114,39 @@ def test_invalid_json_neg_inf(self):
 
 class TestBigQuerySource(unittest.TestCase):
 
+  def test_display_data_item_on_validate_true(self):
+    source = beam.io.BigQuerySource('dataset.table', validate=True)
+
+    nspace, dd = make_nspace_display_data(source)
+    expected_items = [
+        ItemMatcher.matches_kvn('validation', True, nspace),
```

Comment:
Display data tests tend to live among other tests within a component test class. As such, the imported `ItemMatcher` class isn't obviously display-data-specific in context. Is there a better name that makes it contextually obvious that `ItemMatcher` is specific to display data?  Perhaps `DisplayDataMatcher.item_matches(..)`?